### PR TITLE
Node details spinner Chrome display bug fix

### LINKED ISSUE #2173: Details panel spinner looks odd
See video: http://recordit.co/U86TtCFcbA

### PR DESCRIPTION
**Pull request by fbarl**
Fixes #2173.

### CODE REVIEW

File: client/app/scripts/components/node-details.js [hunk 1]
```diff
@@ -73,6 +74,10 @@ class NodeDetails extends React.Component {
   renderLoading() {
     const node = this.props.nodes.get(this.props.nodeId);
     const label = node ? node.get('label') : this.props.label;
+    // NOTE: If we start the fa-spin animation before the node details panel has been
+    // mounted, the spinner is displayed blurred the whole time in Chrome (possibly
+    // caused by a bug having to do with animating the details panel).
+    const spinnerClassName = classNames('fa fa-circle-o-notch', { 'fa-spin': this.props.mounted });
```
Comment (jpellizzari) [inline client/app/scripts/components/node-details.js]: Should we be checking that the `<NodeDetails />` component is mounted instead of its parent?

Comment (fbarl) [inline client/app/scripts/components/node-details.js]: For one thing, it's the `details-card` that gets animated and that animation was supposedly causing the problem, but also I didn't want to copy the mounting logic to `node-details` and having it in two places. Your suggestion seems more *reasonable* somehow, so I'm not sure which approach I like better in the end. I'd say it doesn't matter though, as this code will be gone soon as we switch to @bowenli's new spinners.

Comment (jpellizzari) [inline client/app/scripts/components/node-details.js]: Ok that makes sense. No sense adding state to node-details then.

**Comment by jpellizzari** [comment]
Sorry for the delay on this review @fbarl. LGTM

**Pull request merged**